packages/extension-api/src/extension-api.d.ts

```diff
@@ -248,6 +248,7 @@ declare module '@podman-desktop/api' {
   export interface Link {
     title: string;
     url: string;
+    group: LinkGroupName;
```

Comment:
shouldn't the group be optional ?

Reply:
probably no group name then it's displayed on the dashboard. It's OK I think to have a default group if not specified

Reply:
updated, now the group name is optional